Resources/Prototypes/_Sunrise/Maps/kettle.yml

```
@@ -12,7 +12,7 @@
           nameGenerator:
             !type:NanotrasenNameGenerator
             prefixCreator: '14'
-        - type: StationEmergencyShuttle
+        - type: StationEmergencyShuttle # TODO: Нужен другой шаттл, этот удален
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)